content/docs/tips-and-tricks/azure-guide.md

```diff
@@ -0,0 +1,348 @@
+    "managedByTenants": [],
+    "name": "<YourSubscriptionName>",
+    "state": "Enabled",
+    "tenantId": "<YourTenantId>",
+    "user": {
+      "name": "<YourUserName>",
+      "type": "user"
+    }
+  }
+]
+```
+
+You will need `<YourSubscriptionId>` and `<YourTenantId>` later.
+
+If you don't already have a `Resource group` you would like to use, create a new one:
+
+```bash
+az group create --name "bank-vaults-test-rg" --location "EastUS"
+{...}
+```
+
+### Create an AKS cluster
+
+```bash
+# create cluster
+az aks create --resource-group "bank-vaults-test-rg" --name "bank-vaults-test-cluster" --generate-ssh-keys
+{...}
+
+# write credentials to kubeconfig
+az aks get-credentials --resource-group "bank-vaults-test-rg" --name "bank-vaults-test-cluster"
+
+# if you need to look at cluster information again
+az aks show --resource-group "bank-vaults-test-rg" --name "bank-vaults-test-cluster"
+```
+
+### Create an App Registration and a Client secret
+
+You will use this `App Registration` as the resource for generating MSI access tokens for authentication. A more detailed guide for this can be found [here](https://learn.microsoft.com/en-us/azure/healthcare-apis/register-application-cli-rest).
+
+```bash
+# create App Registration and only return with its Application Id
+az ad app create --display-name "bank-vaults-test-ar" --query appId --output tsv
+<YourAppRegistrationApplicationId>
+
+# create Service Principal for your App Registration
+az ad sp create --id "<YourAppRegistrationApplicationId>" --query id --output tsv
+<YourEnterpriseApplicationObjectID>
+
+# create secret
+az ad app credential reset --id "<YourAppRegistrationApplicationId>" --append --display-name "bank-vaults-test-secret" --query password --output tsv
+# The output includes credentials that you must protect. Be sure that you do not include these credentials in your code or check the credentials into your source control. For more information, see https://aka.ms/azadsp-cli
+<YourAppRegistrationClientSecret>
+
+# authorize the Service Principal to read resources in your Resource Group
+az role assignment create --assignee "<YourEnterpriseApplicationObjectID>" --scope "/subscriptions/<YourSubscriptionId>/resourceGroups/MC_bank-vaults-test-rg_bank-vaults-test-cluster_eastus" --role Reader
+{...}
+```
+
+### Create a Key Vault and permit the AKS cluster to access it
+
+```bash
+# create Azure Key Vault
+az keyvault create --resource-group "bank-vaults-test-rg" --name "bank-vaults-test-kv" --location "EastUS"
+{...}
+
+# get our AKS cluster's Object ID
+az aks show --resource-group "bank-vaults-test-rg" --name "bank-vaults-test-cluster" --query "identityProfile.kubeletidentity.objectId" --output tsv
+<YourAKSClusterObjectID>
+
+# set policy
+az keyvault set-policy --name "bank-vaults-test-kv" --object-id <YourAKSClusterObjectID> --secret-permissions all --key-permissions all --certificate-permissions all
+{...}
+```
+
+### Create Storage account and a Container
```

Comment:
```suggestion
### Create Storage account and a container
```

Reply:
It is also an Azure resource and I tended to write those capitalized

Reply:
then lets remove make it `### Create Storage account and Container`